src/applications/static-pages/sass/modules/_m-facilities.scss

```diff
@@ -41,7 +41,7 @@ $duo-tone-lighten: #000;
     height: 0%;
   }
 
-  /* Cheat for IE10+ not handling flex box correctly.
+  /* Cheat for IE10+ not handling flex box correctly. */
```

Comment:
Oooh, tricky! Not sure how we would have caught this, either.

Reply:
I was surprised the linter didn't catch it.